src/scripts/i18n/i18n_extraction.py

```
@@ -0,0 +1,358 @@
+        "contents": [{
+            "role": "user",
+            "parts": [{"text": prompt}]
+        }]
+    }
+    last_exception = None
+    for _ in range(3):  # Simple retry for transient failures
+        try:
+            response = requests.post(GEMINI_API_URL, json=body, timeout=1200)
+            response.raise_for_status()
+            return response.json()
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)